front/src/app/todo-list/todo-list.component.scss

```diff
@@ -27,6 +47,9 @@
     &:hover {
       background-color: rgb(216, 213, 213);
     }
+    &:hover .task-list__showing-control-menu, .task-list__rename {
+      visibility: visible;
+    }
   }

```

Comment:
I'm confused to see `.task-list__showing-control-menu` and `.task-list__rename` stlyes on hover here while on ` &:focus-within` inside `.task-list &_link`

Please group them together